libs/design/src/atoms/form/qty-field/qty-text/qty-text.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, Input, ViewChild, ElementRef, ChangeDetectionStrategy, ChangeDetectorRef, Optional, Self } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR, NgControl } from '@angular/forms';
+import { DaffInputComponent } from '../../input/public_api';
+import { coerceNumberProperty } from '@angular/cdk/coercion';
+import { DaffFormFieldControl } from '../../form-field/public_api';
+
+@Component({
+  selector: 'daff-qty-text',
+  templateUrl: './qty-text.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [
+    {provide: DaffFormFieldControl, useExisting: DaffQtyTextComponent}
+  ]
+})
+export class DaffQtyTextComponent implements ControlValueAccessor, DaffFormFieldControl<number> {
+ 
+  constructor(
+    @Optional() @Self() public ngControl: NgControl, 
+    private changeDetectorRef: ChangeDetectorRef
+  ) {
+    if (this.ngControl != null) {
+      this.ngControl.valueAccessor = this;
+    }
+  }
+
+  @ViewChild(DaffInputComponent) input : DaffInputComponent;
+
+  /**
+   * @docs
+   * The maximum number for the text field;
+   */
+  @Input() max: number = 10;
+
+  /**
+   * @docs
+   * The minimum number selectable.
+   */
+  @Input() min: number = 1;
```

Comment:
In the view, the `min` attribute is always set to be `1` instead of `min`. Should it be `min` instead?